### ISSUE TITLE
Smarteditor: Upload doesn't work + present files not visible

### ISSUE THREAD

**Issue by jochenwezel** [CompuMasterGmbH/cammIntegrationPortal#36]
In following screenshot, there are several issues visible:
![image](https://cloud.githubusercontent.com/assets/3033827/22520074/2670382a-e8b3-11e6-9873-bd79edd12fc1.png)

Editor tag is configured as follows:
```asp.net
<cammWebEdit:SmartCommonMarkEditor marketlookupmode="0" 
	Images="images" Docs="docs"
	ImagesAllowedFileExtensions="*.jpg,*.gif"
	DocumentsAllowedFileExtensions="*.pdf,*.txt,*.doc,*.docx,*.rtf,*.xls,*.xlsx,*.ppt,*.pptx"
	...
```

1. loremipsum.txt can't be uploaded, error message appears (alternative of ImagesAllowedFileExtensions="jpg,gif" DocumentsAllowedFileExtensions="pdf,txt,doc,docx,rtf,xls,xlsx,ppt,pptx" doesn't work, too)
2. error message shows quotation marks in between
3. documents list is empty, but 2 files .pdf are in the docs directory

**Comment by jochenwezel** [comment]
To point 1: correct form of extensions must be without * (star) but with . (dot) character, e.g. 
ImagesAllowedFileExtensions=".jpg,.gif"

Recommendation 1: send warning to technician in case of found "*.something1,*.something2" or "something1,something2" with recommendation to change into ".something1,.something2"

Recommendation 2: enhanced samples pages to show a valid attribute usage (done separately by @jochenwezel)

Status update to this issue:
- [x] Point 1 origin
- [x] Point 1 recommendation 1
- [x] Point 1 recommendation 2
- [x] Point 2
- [x] Point 3

**Comment by cmalbertschw** [comment]
> 2. error message shows quotation marks in between

Fixed by 0816be16876c557eab0933f91bc102cb024fc55c

**Comment by cmalbertschw** [comment]
> Recommendation 1: send warning to technician in case of found ".something1,.something2" or "something1,something2" with recommendation to change into ".something1,.something2"

It has been decided to support the old formats: "*.something1,*.something2", "something1,something2" and mixed will be converted to ".something1,.something2" 3ccd005098d6e3dc8da6604548643b0fb057de9a
